utils/create-schema-interfaces-interactive.ts

```diff
@@ -1,9 +1,12 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-const { generate } = require('@s1seven/schema-tools-generate-interfaces');
-const path = require('path');
-const readline = require('readline');
-const { stdin: input, stdout: output } = require('process');
-const fs = require('fs');
+/* eslint-disable no-console */
+import fs from 'fs';
+import path from 'path';
+import { stdin as input, stdout as output } from 'process';
+import readline from 'readline';
+
+// TODO: fix module not found error when called from packages/generate-coa-pdf-template/project.json
+// could this be moved to packages/generate-coa-pdf-template/utils?
+import { generate } from '@s1seven/schema-tools-generate-interfaces';
```

Comment:
I did not encounter an issue with the import, but the real problem is that the default Nx executor (nx:run-commands) does not support interactive CLI.
So this should be run directly via ts-node.